src/Pods/Blocks/Types/View.php

```diff
@@ -126,7 +126,20 @@ public function render( $attributes = [] ) {
 
 		// Prevent any previews of this block.
 		if ( is_admin() || wp_is_json_request() ) {
-			return __( 'No preview is available for this Pods View, you will see it on the frontend.', 'pods' );
+			ob_start();
+			?>
+				<div class="pods-block-placeholder_container">
+					<div class="pods-block-placeholder_content-container">
+						<img src="<?php echo esc_url( PODS_URL ); ?>ui/images/pods-logo-green.svg" alt="Pods logo" class="pods-logo">
+						<div class="pods-block-placeholder_content">
+							<h2 class="pods-block-placeholder_title">View</h2>
+							<p><?php echo __( 'No preview is available for this Pods Form, you will see it on the frontend.', 'pods' ); ?></p>
```

Comment:
* :no_entry_sign: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found '__'.
* :warning: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')'

<sub>hash: 1f29064d02bada6d41ad00389af972b4</sub>